x/dsmr/block.go

```diff
@@ -105,6 +106,47 @@ func ParseChunk[T Tx](chunkBytes []byte) (Chunk[T], error) {
 	return c, c.init()
 }
 
+// ExecutionBlock bridge the gap between the dsmr's block implementation and the validity window's execution block interface.
+type ExecutionBlock struct {
+	innerBlock Block
```

Comment:
I think it would be fine to just embed `Block` here since we're just wrapping the data to satisfy the interface impl